packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -16,55 +16,90 @@ type RunOptions = {
 	cwd?: string;
 };
 
+type PrintOptions<T> = {
+	promise: Promise<T> | (() => Promise<T>);
+	useSpinner?: boolean;
+	startText: string;
+	doneText?: string;
+};
+
 export const runCommand = async (
 	command: string,
 	opts?: RunOptions
 ): Promise<string> => {
-	const s = spinner();
+	return printAsyncStatus({
+		useSpinner: opts?.silent, // only use the spinner if we're not spawning with stdio: 'inherit'
```

Comment:
As we discussed yesterday, how the spinner acts in tests has been moved into the spinner function. No need to consider it outside.

So in this case, we disable the spinner when the child_process inherits stdin/stdout because they don't play nice together